style.css

```diff
@@ -450,3 +474,290 @@ textarea:invalid:required {
+    font-weight: 500;
+  }
+
+  .foot-connect {
+    margin-left: 6em;
+  }
+
+  .foot-connect ul {
+    max-width: 20%;
+  }
+
+  .grid-container {
+    margin-bottom: 6rem;
+    margin-left: 3rem;
+  }
+
+  .template-wrapper {
+    margin-right: 5rem;
+  }
+
+  .wrapping {
+    display: flex;
+  }
+
+  .mobile-image {
+    display: none;
+  }
+
+  .desk-image {
+    display: block;
+  }
+
+  .desktop-tonic {
+    margin: 24px;
+    padding-left: 25px;
+    padding-right: 50px;
+  }
+
+  .Tonic h2 {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
+  .Tonic-list ul {
+    padding-bottom: 15px;
+  }
+
+  .Tonic-content p {
+    padding-bottom: 10px;
+  }
+
+  .Tonic-link ul {
+    padding-bottom: 20px;
+  }
+
+  .two {
+    order: 1;
+  }
+
+  .canopy {
+    font-weight: 500;
+    font-size: 18px;
+  }
+
+  .canopy-list {
+    font-weight: 400;
+    font-size: 18px;
+  }
+
+  .Tonic-content {
+    font-size: 16px;
+  }
+
+  .Ruby {
+    display: block;
+  }
+
+  .disable {
+    color: #6070ff;
+    border: 1px solid #7f8cff;
+    pointer-events: fill;
+    cursor: default;
+  }
+
+  .disable:hover {
+    background-color: #6070ff;
+    color: #fff;
+    text-decoration: underline;
+  }
+
+  .disable:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .action-button {
+    display: flex;
+  }
+
+  .three {
+    order: 1;
+  }
+
+  .wrap-footer {
+    display: flex;
+  }
+
+  .wrap-footer-one {
+    margin-left: 4rem;
+    margin-right: 0;
+    flex-basis: 500px;
+  }
+
+  .flex-connect-three {
+    font-size: 16px;
+    font-weight: 500;
+    color: #7f8cff;
+  }
+
+  .flex-connect-three h3 {
+    margin-left: 0;
+  }
+
+  .wrap-footer-two {
+    flex-basis: 700px;
+    margin-left: 2rem;
+  }
+
+  .language ul {
+    margin-top: 7rem;
+    justify-content: space-between;
+    margin-right: 6rem;
+  }
+
+  .language p {
+    font-size: 30px;
+    font-weight: 600;
+  }
+
+  .lang-wrapper {
+    margin-left: 2rem;
+  }
+
+  .lang2 {
+    display: inline-block;
+    margin: 0 5px;
+    padding-left: 3px;
+    padding-right: 30px;
+  }
+
+  .Frameworks1 ul {
+    margin-right: 6rem;
+  }
+
+  .Skills ul {
+    margin-right: 6rem;
+  }
+
+  footer {
+    margin: 0;
+  }
+
+  .resume {
+    margin: 15px;
+  }
+
+  .lang2 ul {
+    display: block;
+  }
+
+  .contact-details {
+    background-image: url(./desktop-contactimage.png);
+    display: flex;
+    justify-content: center;
+    padding-bottom: 4rem;
+  }
+
+  form {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact-content {
+    padding-bottom: 10px;
+  }
+
+  .contact-content p {
+    font-size: 20px;
+    font-weight: 300;
+    text-align: center;
+  }
+
+  input {
+    padding-right: 20rem;
+    border-radius: 0;
+  }
+
+  textarea {
+    border-radius: 0;
+    padding-right: 20rem;
+  }
+
+  .submit {
+    padding: 1rem 1rem;
+    font-size: 17px;
+    font-weight: 500;
+  }
+
+  .divide {
+    display: none;
+  }
```

Comment:
- [ ] Kindly also make sure to fix mobile version text fields too.
- Your design:
<img width="369" alt="Screenshot 2022-03-23 at 9 25 25 PM" src="https://user-images.githubusercontent.com/70269507/159770658-212fcffe-c12c-4cce-aec6-9f28025f4bfe.png">
- Figma design:
<img width="349" alt="Screenshot 2022-03-23 at 9 25 06 PM" src="https://user-images.githubusercontent.com/70269507/159770768-64ba06b3-8713-4b19-b7ae-c5032e66d452.png">